tests/CarrierTest.php

```diff
@@ -69,6 +69,16 @@ public function testRoyalmailClassRealValues()
     }
 
     /**
+     * Test to ensure that the calculate method class is returning rates with
+     * the ignore insurance flag set to true.
+     */
+    public function testRoyalmailClassRealValuesAll()
+    {
+        $this->assertNotEmpty($this->carrier->getRates('GB', 20, 0.050, true));
```

Comment:
Would it have received a non-empty result if it was false too? It would be good to have a bit more of a test here.